### ISSUE TITLE
How to play songs continuously?

### ISSUE THREAD

**Issue by memmartins** [matt1/CLIMediaRenderer#5]
Hello,

I'm using cliMediaRenderer with serviio on Ubuntu 12.04 and I have tested several control points on Android devices, such as MediaHouse, UpnPlay and BubbleUpnP.
In general, most features are working fine, but I can't get the queued songs to play continuously, one after another. After a song finish playing, I always need to press the "next" button to start the next song.
Is there a solution?
Another important feature would be the ability to control renderer volume.

Thanks!

**Comment by xunxky** [comment]
yes it is quite a good system however it misses several control features
